Packages/Web/lib/util.flp

```diff
@@ -459,3 +459,46 @@ end
 func trackName(propertyName, id)
   return "\(id)_\(propertyName.snakeCased())Track"
 end
+
+func trackOffsets(_ times, _ duration)
+    var offsets = []
+    if times[0] != 0
+        offsets = [0]
+    end
+    for time in times
+        offsets = offsets.appending(time / duration)
+    end
+    if offsets[offsets.count - 1] != 1
+        offsets = offsets.appending(1)
+    end
```

Comment:
I think we can remove this if statement now that were using simulated key values